app/controllers/api/v1/visitor_details_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::VisitorDetailsController < AuthenticatedApiController
+  def show
+    @assignments = VisitorLookup.new(index_params).visitor.assignments.includes(:variant_details)
```

Comment:
a bit of a law-of-demeter violation on this one, but it's not killing anyone.

i think that if this "visitor detail" concept grows at all then we will wanna spawn a little model for this that can internalize these nuanced access patterns bits -- i.e. the surprising includes call and the defaulting that we do inside of Assignment for a non-existent variant detail record.